tests/singlylinkedlist_ut/main.c

```diff
@@ -1,3 +1,4 @@
+#include <stddef.h>
```

Comment:
missing copyright

Reply:
Fixed.

---
In reply to: [291777946](https://github.com/Azure/azure-c-shared-utility/pull/331#discussion_r291777946) [](ancestors = 291777946)